File: 2215-find-the-difference-of-two-arrays/2215-find-the-difference-of-two-arrays.py
class Solution(object):
    def findDifference(self, nums1, nums2):
        """
        :type nums1: List[int]
        :type nums2: List[int]
        :rtype: List[List[int]]
        """
        l1 = [x for x in nums1 if x not in nums2]
        l2 = [x for x in nums2 if x not in nums1]
        return [list(set(l1)), list(set(l2))]

File: 2215-find-the-difference-of-two-arrays/2215-find-the-difference-of-two-arrays.go
func findDifference(nums1 []int, nums2 []int) [][]int {
    m1, m2 := map[int]bool{}, map[int]bool{}
    for _, v := range nums1 {
        m1[v] = true
    }

    for _, v := range nums2 {
        m2[v] = true
    }

    ans1 := []int{}
    for _, v := range nums1 {
        if !m2[v] {
            ans1 = append(ans1, v)
        }

        m2[v] = true
    }
    
    ans2 := []int{}
    for _, v := range nums2 {
        if !m1[v] {
            ans2 = append(ans2, v)
        }

        m1[v] = true
    }

    return [][]int{ans1, ans2}
}